test/include/test_helper/test_helper.h

```diff
@@ -23,13 +23,17 @@ struct TestQueryConfig {
 
 class TestHelper {
 public:
-    static constexpr char E2E_TEST_FILES_DIRECTORY[] = TEST_FILES_DIR;
+    inline static std::string E2E_TEST_FILES_DIRECTORY = "test/test_files";
```

Comment:
remove inline

Reply:
removed

Reply:
If we remove the inline here, this static variable would be required to be defined somewhere else outside of this class. I think that would reduce the readability of the code? Define this variable with inline here seems to be more compatible with our current code design.